### PR TITLE
Console error fix

### PR DESCRIPTION
**Pull request by pindge**
Change `jupyterlab` base version to 1.2.3 - tested compatability with `jupyterlab-git`
Changer `jupyterlab-code-formatter` to `jupyter_code_fomatter` as per official doc
https://jupyterlab-code-formatter.readthedocs.io/en/latest/index.html

```
jupyter labextension install @ryantam626/jupyterlab_code_formatter
pip install jupyterlab_code_formatter
jupyter serverextension enable --py jupyterlab_code_formatter
```

### CODE REVIEW

File: docker/requirements.txt [hunk 1]
```diff
@@ -53,13 +53,13 @@ urbanaccess
 
 # jupyter things
 jupyter==1.0.0
-jupyterlab==1.0.9
+jupyterlab==1.2.3
 jupyterhub==0.9.6
```
Comment (tom-butler) [inline docker/requirements.txt]: can we increase version of jupyterhub too? the hub is using 1.1.0 and I believe they should match?

File: docker/requirements.txt [hunk 2]
```diff
@@ -53,13 +53,13 @@ urbanaccess
...
 ipyleaflet==0.11.1
 jupyter-server-proxy==1.1.0
 dask-labextension==1.0.3
 nbdime==1.1.0
-jupyterlab-code-formatter==0.5.0
+jupyterlab_code_formatter==0.5.0
```
Comment (tom-butler) [inline docker/requirements.txt]: not sure about this change the python package is at https://pypi.org/project/jupyterlab-code-formatter/ (without underscores)

File: docker/requirements.txt (hunk 1 above)
Comment (pindge) [reply docker/requirements.txt]: `jupyterlab==1.2.3` was released Nov 13 2019, `jupyterhub==1.1.0` was released jan 17, 2020.

I will test this first.

File: docker/requirements.txt (hunk 2 above)
Comment (pindge) [reply docker/requirements.txt]: let me test with the `-`

Comment (Kirill888) [reply docker/requirements.txt]: it doesn't matter for python package names, so just revert that change, should work

**Comment by pindge** [comment]
Tested `jupyterhub==1.1.0`

**Pull request merged**